sql/load_sql_context.sql

```diff
@@ -192,6 +221,10 @@ select
                                 'max_rows', coalesce(
                                     (graphql.comment_directive(pg_catalog.obj_description(pn.oid, 'pg_namespace')) ->> 'max_rows')::int,
                                     30
+                                ),
+                                'resolve_base_type', coalesce(
+                                    (graphql.comment_directive(pg_catalog.obj_description(pn.oid, 'pg_namespace')) -> 'resolve_domain_types') = to_jsonb(true),
+                                    true
```

Comment:
Shouldn't the default be `false`?

Reply:
I thought we were going with this minor change you have to opt out of but I have no issue making it opt into.

Reply:
On second thought I think you're right, 2.0 it can change to the default.